__tests__/database/migration/file/index.test.ts

```diff
@@ -7,6 +7,6 @@ describe('TESTING all migration file', () => {
     // Execution
 
     // Assertion
-    expect(allMigrations.length).toEqual(18);
+    expect(allMigrations.length).toEqual(22);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The number of migrations has been increased from 18 to 22. Ensure that the four new migrations have been properly implemented and tested. If they are not correctly done, it could lead to data loss or corruption.
```diff
-    expect(allMigrations.length).toEqual(18);
+    expect(allMigrations.length).toEqual(22);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->